### PR TITLE
COMMERCE-7176 Both Uuid and CPInstanceUuid need to be unique.

### PR DESCRIPTION
**Pull request by brianikim**
https://issues.liferay.com/browse/COMMERCE-7176

@andrea-ale-sbarra

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLocalServiceImpl.java [hunk 1]
```diff
@@ -724,6 +724,7 @@ public CPDefinition copyCPDefinition(long cpDefinitionId, long groupId)
 
 			newCPInstance.setUuid(PortalUUIDUtil.generate());
 			newCPInstance.setCPInstanceId(counterLocalService.increment());
+			newCPInstance.setCPInstanceUuid(PortalUUIDUtil.generate());
```
Comment (marco-leo) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLocalServiceImpl.java:727]: @brianikim this should already happen automatically

Comment (brianikim) [reply modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLocalServiceImpl.java:727]: hey @marco-leo , it does not. The uuid is still the same for both original and copy without that inclusion

**Comment by christian-chiappa** [comment]
https://github.com/liferay-commerce/liferay-portal/pull/2104

**Pull request closed without merging**